src/report/report-list.controller.js

```diff
@@ -64,11 +64,16 @@
             jasper: getJasperReports('ADMINISTRATION'),
             superSet: getSuperSetReports('ADMINISTRATION')
         };
+        var dashboardReports = {
+            jasper: getJasperReports('DASHBOARD'),
+            superSet: getSuperSetReports('DASHBOARD')
+        };
 
         console.log('stockReports', stockReports);
         console.log('requisitionReports', requisitionReports.superSet);
         console.log('orderReports', orderReports);
         console.log('administrationReports', administrationReports.superSet);
+        console.log('dashboardReports', dashboardReports.superSet);
```

Comment:
Is this and other console.logs needed here? Console.logs and other debugging tools shouldn't be really committed to production. If it's not necessary here I would suggest to remove it

Reply:
We need to see the output in the console if everything works properly on the uat instance since we cannot test superSets locally, but we should delete them later for sure